### PR TITLE
Generate compiled files.

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js please review.

generating compiled files.

### CODE REVIEW

**Comment by wytrych** [comment]
It would be very nice to have the `ngtemplates` as a grunt plugin. It could go to https://github.com/incuna/grunt-incuna-plugins

**Comment by grahamgilchrist** [comment]
`ngtemplates` is already a third party grunt plugin?

**Comment by grahamgilchrist** [comment]
although that bit to iterate over the module directories and return the config is repeated in lots of packages, so it would be nice to package that

**Comment by grahamgilchrist** [comment]
Looks good to merge when tests pass

**Comment by Minglee01** [comment]
if a jira case is made im happy to do it later

**Comment by Minglee01** [comment]
as an enhancement.

**Comment by wytrych** [comment]
Yes that's what I ment, and maybe the options could be there too, as we don't change them from project to project.

File: Gruntfile.js [hunk 1]
```diff
@@ -11,6 +34,23 @@ module.exports = function (grunt) {
 
     require('time-grunt')(grunt);
 
+    var concatConfig = {
+        target: {
+            src: [
+                'modules/**/**/*.js'
```
Comment (henrahmagix) [inline Gruntfile.js:40]: Why so many `**`? Just one set should do

File: Gruntfile.js [hunk 2]
```diff
@@ -23,6 +63,16 @@ module.exports = function (grunt) {
                 ]
             }
         },
+        watch: {
+            templates: {
+                files: 'modules/**/*.html',
+                tasks: 'ngtemplates'
+            },
+            scripts: {
+                files: 'modules/**/scripts/**/*.js',
```
Comment (henrahmagix) [inline Gruntfile.js:72]: Should this be specific as `modules/*/scripts/...`?

**Comment by henrahmagix** [comment]
👍

**Pull request merged**